### ISSUE TITLE
Is it possible to run this locally?

### ISSUE THREAD

**Issue by Ctesias** [ashiswin/Palworld-Dedicated-Server-Host-Fix#1]
Can I run the transfer script with the local and server player files from my local machine and then upload the fixed player saves and the modified level.sav to the dedicated server?

I ask because I don't have CLI access to my Nitrado hosted server.

**Comment by ashiswin** [comment]
Yes absolutely, you can do it locally too :)

**Comment by ow1337** [comment]
Did u test it locally @ashiswin ? So i tested out from my local PC, like i did before on my own local machine, exact same way for  a new server, it didn't work, my Friend who has the new server runned the script and my player file is back.

**Comment by ashiswin** [comment]
Yep, most testing was done locally before my server arrived for me to host it

**Comment by ow1337** [comment]
Then i messed something up after i did 3x times successfully :D But now i should have a savefile who is useable for all servers.. On my local laptop machine i got a different ID, what was also weird. Still thanks for your tool!

**Comment by ow1337** [comment]
@Ctesias did u got it running on nitrado? everybody i know right now said nitrado seems not to work for savegames

**Comment by Ctesias** [comment]
I got the map/server to run, but I am struggling with the user saves.

I saw a reddit comment that a -nosteam flag in the server startup command
might be causing users to get new player files once they're on the server,
I have a ticket open with nitrado to test.

**Comment by Ctesias** [comment]
@ow1337 I have no been able to get the characters from my local game to load in the nitrado dedicated server despite running the script.  

Anyone have suggestions on a hosting platform that this does work on?
